core/trino-main/src/main/java/io/trino/operator/MultiChannelGroupByHash.java

```diff
@@ -682,11 +682,15 @@ public Void getResult()
     }
 
     class AddLowCardinalityDictionaryPageWork
-            extends LowCardinalityDictionaryWork<Void>
+            implements Work<Void>
     {
+        private final Page page;
+        private int[] combinationIdToPosition;
+        private int nextCombinationIndex;
```

Comment:
`nextCombinationIndex` -> `nextCombinationId` since key to `combinationIdToPosition` array is `combination id` and not `index`